pkg/kf/commands/install/gke/gke.go

```diff
@@ -431,6 +445,22 @@ func gkeClusterConfig(
 		}
 	}
 
+	// Machine Type
+	{
+		availableMachineTypes, err := machineTypes(ctx, projID, cfg.zone)
+		switch {
+		case err != nil:
+			return gkeConfig{}, err
+		case len(availableMachineTypes) == 1:
+			cfg.machineType = availableMachineTypes[0]
+		default:
+			_, cfg.machineType, err = SelectPrompt(ctx, "Machine Type", availableMachineTypes...)
```

Comment:
nit: We don't really do this elsewhere, but I'm wondering if we should change the title to "Machine Type (recommended 'n1-standard-4')". WDYT?